events-api/protobuf/v1/event.proto

```diff
@@ -67,3 +69,33 @@ enum Version {
   UnknownVersion = 0;
   V1 = 1;
 }
+
+// Event specific details
+message EventDetails {
+  // Rebuild event details
+  optional RebuildEventDetails rebuild_details = 1;
+}
+
+// Rebuild event details
+message RebuildEventDetails {
+  // Rebuild end status
+  optional RebuildEndStatus rebuild_end_status = 1;
```

Comment:
Is this optional for the case of start rebuild?
In that case I suggest renaming RebuildEndStatus to RebuildStatus and adding a Started there.